boards/DCM/Src/App/App_Imu.c

```diff
@@ -0,0 +1,49 @@
+#include <assert.h>
+#include <stdlib.h>
```

Comment:
Alright app code looking good 👍👍👍

Reply:
Thanks, tbh learning Git was the hardest part